images/macos/software-report/SoftwareReport.Java.psm1

```diff
@@ -10,10 +10,11 @@ function Get-JavaVersions {
         $javaPath = $_.Value
         # Take semver from the java path
         $version = $javaPath.split('/')[5]
+        $fullVersion = $version.Replace('-', '+')
```

Comment:
I think that it should be like:
$fullVersion = $version.Replace('+', '-')

Reply:
Or do we want to state `+` in docs?

Reply:
In the path it should be like `8.0.282-8`, but for software report version should be as `8.0.282+8`